crates/typos-dict/tests/verify.rs

```diff
@@ -7,7 +7,7 @@ use unicase::UniCase;
 type Dict = BTreeMap<UniCase<String>, IndexSet<String>>;
 
 #[test]
-fn verify() {
+fn words_csv() {
```

Comment:
Have a look at the last commit message. Just running `cargo test` fails because the codegen test fails on duplicate CSV entries ... so you firstly have to overwrite the words CSV before you can overwrite the generated code. I think that is much more clearer when the test is named words_csv ... and not just "verify".

Reply:
I disagree and,. as I said, this is independent of this change.  We can discuss improvements to that workflow elsewhere.

Reply:
Fair enough ... dropped it.